test/test_nn_ops.cpp

```diff
@@ -588,7 +588,7 @@ TEST(NNOps, Conv2DBackpropInputNCHWSame) {
   opexecuter_tf.ExecuteOnTF(tf_outputs);
 
   // Compare NGraph and TF Outputs
-  Compare(tf_outputs, ngraph_outputs);
+  Compare(tf_outputs, ngraph_outputs, 1e-05, 1e-05);
```

Comment:
Are they not passing with the default values ? 1e-05, 1e-08 ?

Reply:
Yes, I ran the whole tests for ~50 runs, for some run, those nn ops fail, so I changed them to a looser tolerance.

One thing to notice is that even with 1e-05 and 1e-05, they are more strict than our previous Compare method I believe because according to our formula, if rtol = 0, the atol is 1e-05, so abs(desired - actual) <= 0.00001, which previously is 0.001.